src/proxies/InverterBeacon_v1.sol

```diff
@@ -63,10 +63,10 @@ contract InverterBeacon_v1 is IInverterBeacon_v1, ERC165, Ownable2Step {
     ) {
         if (
             // Minor Version cant go down
-            newMinorVersion < minorVersion
             // Patch Version cant go down or stay the same if minorVersion stays the same
-            || newPatchVersion <= patchVersion
-                && newMinorVersion == minorVersion
+            newMinorVersion < minorVersion
+                || newPatchVersion <= patchVersion
+                    && newMinorVersion == minorVersion
```

Comment:
What happened here? Your version looks fine, but interesting that the "before" part looks so weird...

Reply:
Oh I see, you just didn't like the formatting, the version before was fine, just had comments within the expression :D Or did you not change this? @0xNuggan

Reply:
Oh it formats it the same for me locally after updating my Foundry, ignore this 😂

Reply:
That was the formatter. The comments were originally inlined directly inside the if() statement, which probably goes against some rule.